mqtt-launcher.py

```diff
@@ -130,7 +133,7 @@ def on_disconnect(mosq, userdata, rc):
     transportType = cf.get('mqtt_transport_type', 'tcp')
 
     # initialise MQTT broker connection
-    mqttc = paho.Client(clientid, clean_session=False, transport=transportType)
+    mqttc = paho.Client(paho.CallbackAPIVersion.VERSION2, clientid, clean_session=True, transport=transportType)
```

Comment:
Why did you change `clean_session` from False as default to True?

Reply:
Was testing with some options and forgot to change it back. False is fine.

Reply:
If you change that back, I’ll gladly commit your PR.

Reply:
my be a dumb questin, but i'm not used to github.... can i edit it and the pull will update or do i have to mak a new request?